index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <head>
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-*********************" crossorigin="anonymous">
+      </head>
+      
+    <title>Portfolio</title>
+</head>
+<body>
+   <div class="top-navigation">
+    <nav class="nav-bar">
+        <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+   </div>
```

Comment:
Nice work setting up your tool bar😉 but I think we can make this even better by wrapping our toolbar section in a `header` tag as this is more semantic and will aid user accessibility for screen readers, and will make your code easily maintainable and understandable🤗🤗 Also, your toolbar looks a bit different from the given design😉😉

yours
<img width="319" alt="your toolbar" src="https://user-images.githubusercontent.com/64643995/219600899-bd005026-1180-4374-b348-34bee4af3a48.png">

design
<img width="284" alt="design_nav" src="https://user-images.githubusercontent.com/64643995/219600968-ebe115e9-6ba9-4c7b-9cf2-d415663bd298.png">

Kindly fix this, as it is a requirement that we stick to the design in this milestone🤗🤗